frontend/src/components/core/StreamlitDialog/StreamlitDialog.tsx

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-import CopyToClipboard from "react-copy-to-clipboard"
```

Comment:
Was there a reason we wanted to stop using the react version of this library?

Reply:
It started as a way to make TypeScript happy, but as I explored, I found the library to be really simple (See bt/react-copy-to-clipboard/blob/master/src/Component.js#L35). So I figured I reduced one (albeit small) dependency)